src/more-infos/more-info-content.html

```diff
@@ -35,11 +35,14 @@
   },
 
   stateObjChanged: function (stateObj) {
-    if (!stateObj) return;
-
-    window.hassUtil.dynamicContentUpdater(
-      this, 'MORE-INFO-' + window.hassUtil.stateMoreInfoType(stateObj).toUpperCase(),
-      { hass: this.hass, stateObj: stateObj });
+    if (!stateObj) {
+      window.hassUtil.dynamicContentUpdater(
+        this, null, { isVisible: false });
```

Comment:
by setting it to null, if we close a more info light and open it again, it has to re-create all the DOM. The old logic was faster. 

Also, no need to pass attributes if you're not going to render anything?

I think that it might make more sense to just check the lastChild, and if it exists set `lastChild.isVisible = false`